webview/src/plots/components/comparisonTable/cell/ComparisonTableBoundingBoxImg.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react'
+import {
+  ComparisonBoundingBoxClasses,
+  ComparisonPlotBoundingBox
+} from 'dvc/src/plots/webview/contract'
+import { ComparisonTableBoundingBoxColorFilter } from './ComparisonTableBoundingBoxColorFilter'
+import styles from '../styles.module.scss'
+
+export const ComparisonTableBoundingBoxImg: React.FC<{
+  src: string
+  boxCoords: ComparisonPlotBoundingBox[]
+  classes: ComparisonBoundingBoxClasses
+  alt: string
+}> = ({ alt, src, boxCoords, classes }) => {
+  const [naturalWidth, setNaturalWidth] = useState(0)
+  const [naturalHeight, setNaturalHeight] = useState(0)
+
+  useEffect(() => {
+    const img = new Image()
+    img.src = src
+
+    img.addEventListener('load', () => {
+      setNaturalWidth(img.naturalWidth)
+      setNaturalHeight(img.naturalHeight)
+    })
+  }, [src])
+
+  return (
+    <svg
+      className={styles.image}
+      viewBox={`0 0 ${naturalWidth} ${naturalHeight}`}
+      aria-label={alt}
+      role="img"
+    >
+      {Object.entries(classes).map(
+        ([label, { color, selected }]) =>
+          selected && (
+            <ComparisonTableBoundingBoxColorFilter key={label} color={color} />
+          )
+      )}
+      <image href={src} width={naturalWidth} height={naturalHeight} />
+      {boxCoords.map(({ label, boxes }) => {
+        const labelColor = classes[label].color
+        return boxes.map(({ h, w, x, y }) => (
+          <React.Fragment key={label + h + w + x + y}>
+            <text
```

Comment:
With the current design, I don't think we need this text as we are repeating a lot of information:

![image](https://github.com/iterative/vscode-dvc/assets/37993418/c122a2cb-11ef-4875-8b55-c0216237dcc9)

I.e. there is a big box that shows traffic light objects are yellow. I would drop this text unless we include the threshold along with the text. 

Once there are a lot of boxes this will become even more crowded:

![image](https://github.com/iterative/vscode-dvc/assets/37993418/6100d7aa-1dc7-450a-8bd3-447c2169db14)

WDYT @dberenbaum?

Reply:
Let's also consider how we can filter the boxes by a provided threshold (i.e. "Objects detected with prediction threshold >= 0.5")

Reply:
> I.e. there is a big box that shows traffic light objects are yellow. I would drop this text unless we include the threshold along with the text.
> Once there are a lot of boxes this will become even more crowded

Good point! I chose to add labels since all classes toggle buttons may not be showing (if there are too many, we're going to add a "Show More" button that shows the hidden classes).  

No strong preference though, I'm happy to remove them!

Reply:
@AlexandreKempf, what do you think about keeping text labels on each individual box or would it be better to remove them?

Reply:
Working on object detection, we get used to overlapping text ^^", so we can keep them it is not a problem.
In my opinion, the best behavior would be to have the image with all the labels when not hovered. When hovered, only the bounding box below the mouse gets its text displayed. It is pretty tricky to get which bounding boxes to select based on mouse location, especially with lots of overlapping cases. So generally, what we end up doing is filtering the bounding boxes based on label or score ;)

Reply:
> In my opinion, the best behavior would be to have the image with all the labels when not hovered. When hovered, only the bounding box below the mouse gets its text displayed. It is pretty tricky to get which bounding boxes to select based on mouse location, especially with lots of overlapping cases. 

Sounds good! I'll add this feature to our todo list.